pages/volunteersignup.tsx

```diff
@@ -0,0 +1,494 @@
+import {
+    Button,
+    Box,
+    Center,
+    Flex,
+    Text,
+    Input,
+    FormControl,
+    FormLabel,
+    Link,
+    Progress,
+    Stack,
+    HStack,
+    VStack,
+    Select,
+    Checkbox,
+    Textarea,
+    Heading,
+    UnorderedList,
+    ListItem,
+    OrderedList,
+    Radio,
+    RadioGroup,
+} from "@chakra-ui/react";
+import { CloseButton } from "@components/CloseButton";
+import { BackButton } from "@components/BackButton";
+import { useState } from "react";
+import { GetServerSideProps } from "next"; // Get server side props
+import { getSession, GetSessionOptions } from "next-auth/client";
+import Wrapper from "@components/SDCWrapper";
+import useLocalStorage from "@utils/useLocalStorage";
+import { roles, locale, province, VolunteerInput } from "@models/User";
+
+const BLUE = "#0C53A0"; // TODO: move to src/styles
+const RADIO_YES = "yes";
+const RADIO_NO = "no";
```

Comment:
looks like `RADIO_YES` and `RADIO_NO` are not used anymore so they can be removed. same with some of the imports I think